testhelpers/db.go

```diff
@@ -151,8 +151,17 @@ func loadFixtureChainFromString(db *sql.DB, fixture string) {
 	if err != nil {
 		panic(err)
 	}
-	for tableName, tableData := range content {
-		InsertBatch(db, tableName, tableData)
+	for _, item := range content {
+		data := make([]map[string]interface{}, 0, len(item.Value.([]interface{})))
+		for _, row := range item.Value.([]interface{}) {
```

Comment:
We need this change because we have to treat fixtures as ordered maps instead of usual Golang maps. Otherwise we cannot guarantee the order of data insertions into different tables.